packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/get-app-id.ts

```diff
@@ -0,0 +1,13 @@
+import { stateManager } from 'amplify-cli-core';
+
+/**
+ * returns the appID of the project if exists
+ */
+export const getAppId = () : string => {
```

Comment:
currently I am using this because context object isn't present in places and I have to send empty context as parameter.

Ill see if I can refractor this